### ISSUE TITLE
ProjectE misleading support for Forestry saplings, enchanted books

### ISSUE THREAD

**Issue by RoyalTurkey1** [xJon/Tekkit-2#120]
**Basic details**
- **Are you using a legitimate Minecraft account?**: yes
- **Are you using Java 8, 64-bit?**:  yes
- **Did you add any additional mods**: no
- **Crash/latest log link (if applicable)**: not applicable

**Describe the issue**

A clear description of what the issue is.

When placing any sapling into a Condenser, instead of duplicating same sapling, turns them into Apple oak which are unplacable

**Comment by xJon** [comment]
Seems like the Energy Condenser is missing some needed compatibility with Forestry's saplings.
<details><summary>When placing:</summary>

```
Description: Error executing task

java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: Nbt has no Genome {}
    at java.util.concurrent.FutureTask.report(FutureTask.java:122)
    at java.util.concurrent.FutureTask.get(FutureTask.java:192)
    at net.minecraft.util.Util.runTask(SourceFile:531)
    at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:723)
    at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668)
    at net.minecraft.server.integrated.IntegratedServer.tick(IntegratedServer.java:185)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
    at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.IllegalArgumentException: Nbt has no Genome {}
    at forestry.arboriculture.genetics.Tree.<init>(Tree.java:76)
    at forestry.arboriculture.genetics.TreeRoot.getMember(TreeRoot.java:178)
    at forestry.arboriculture.items.ItemGermlingGE.onItemRightClick(ItemGermlingGE.java:150)
    at net.minecraft.item.ItemStack.useItemRightClick(ItemStack.java:217)
    at net.minecraft.server.management.PlayerInteractionManager.processRightClick(PlayerInteractionManager.java:358)
    at net.minecraft.network.NetHandlerPlayServer.processTryUseItem(NetHandlerPlayServer.java:765)
    at net.minecraft.network.play.client.CPacketPlayerTryUseItem.processPacket(SourceFile:32)
    at net.minecraft.network.play.client.CPacketPlayerTryUseItem.processPacket(SourceFile:9)
    at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at net.minecraft.util.Util.runTask(SourceFile:529)
    ... 5 more
```
</details>

**Comment by xJon** [comment]
After talking to @ACGaming, I understand the mod won't touch NBT data probably by design to prevent dupes (e.g. with backpacks). It's *technically* possible to hardcode special care for edge cases like this, albeit risky - If someone wants to they can PR over https://github.com/ACGaming/ProjectE.
The workaround will be to just remove EMC values from the special saplings due to the way they hold data.

**Comment by Speiger** [comment]
@xJon There is a whitelist in the mod.
https://github.com/ACGaming/ProjectE/blob/17b2bdb0c7c6a64340e858e743d0388f43c4f136/src/main/java/moze_intel/projecte/utils/NBTWhitelist.java#L20

**Comment by xJon** [comment]
@Speiger Interesting! It seems to be incomplete in 1.12.2 though unfortunately, see [this](https://github.com/sinkillerj/ProjectE/issues/2069#issuecomment-737659394) and [this](https://github.com/sinkillerj/ProjectE/issues/1962#issuecomment-587229239).

**Comment by xJon** [comment]
This is resolved in version `1.1.0` by removing the EMC values from the unsupported items.
